MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -1504,6 +1498,53 @@ private void editUnit(UUID id, int idx, boolean salvage) {
         name.setText(ustatus.getDesc());
     }
 
+    private void showPrisoner(UUID id) {
+        //dialog
+        PrisonerStatus pstatus;
+        pstatus = tracker.getPrisonerStatus().get(id);
+
+        if(null == pstatus || null == pstatus.getPerson() ) {
+            return;
+        }
+
+        PersonViewPanel pvp = new PersonViewPanel(
+                pstatus.getPerson(),tracker.getCampaign(),tracker.getCampaign().getApp().getIconPackage());
+
+        final JDialog dialog = new JDialog(frame, "Prisoner View", true); //$NON-NLS-1$
+        dialog.getContentPane().setLayout(new GridBagLayout());
+        JButton btn = new JButton(Messages.getString("Okay")); //$NON-NLS-1$
+        btn.addActionListener(e -> dialog.setVisible(false));
```

Comment:
To help with accessibility, you can make the OK button the default.
```suggestion
        btn.addActionListener(e -> dialog.setVisible(false));
        dialog.getRootPane().setDefaultButton(btn);
```

Reply:
Sure thing. I can fix it up after work.

Reply:
No worries, I didn't realize this landed already! I was reviewing on my phone.